bftengine/src/preprocessor/RequestProcessingInfo.hpp

```diff
@@ -44,6 +44,7 @@ class RequestProcessingInfo {
   bool isReqTimedOut() const;
   uint64_t getReqTimeoutMilli() const { return clientPreProcessReqMsg_->requestTimeoutMilli(); }
   std::string getReqCid() const { return clientPreProcessReqMsg_->getCid(); }
+  bool isPreProcessReqMsgReceivedInTime(const uint16_t preProcessReqWaitTimeMilli) const;
```

Comment:
why is the `preProcessReqWaitTimeMilli` passed as a parameter for each timeout check? Does it change?
I'd rather imagine it as a member variable of `RequestProcessingInfo`...

Reply:
I've changed the approach now.